src/AppBundle/Controller/ArticleInfoController.php

```diff
@@ -76,9 +77,14 @@ public function resultAction(Request $request)
         $project = $request->attributes->get('project');
         $page = $request->attributes->get('article');
 
-        // sets the database name within $this->lh
-        $dbValues = $this->lh->databasePrepare($project, 'ArticleInfo');
-        $projectUrl = $dbValues['url'];
+        $projectData = ProjectRepository::getProject($project, $this->container);
```

Comment:
We really need to get the variable names uniform - in this case, $project is in use elsewhere.